src/com/sibvisions/kitchensink/samples/containers/SplitPanelSample.java

```diff
@@ -74,4 +118,25 @@ public String getName()
 		return "Split";
 	}
 	
+	/* Creates an {@link ICellEditor} that can be used for editing the
+	 * Orientation of controls.
+	 *
+	 * @return the {@link ICellEditor}.
+	 */
+	public ICellEditor createOrientationCellEditor()
+	{
+		UIEnumCellEditor cellEditor = new UIEnumCellEditor();
+		cellEditor.setAllowedValues(new Object[] {
+				new BigDecimal(ISplitPanel.SPLIT_LEFT_RIGHT),
+				new BigDecimal(ISplitPanel.SPLIT_TOP_BOTTOM)
+				
+		});
+		cellEditor.setDisplayValues(new String[] {
+				"Left/Right",
+				"Top/Bottom"
+		});
+		
+		return cellEditor;
+		

```

Comment:
This blank line should be after the function, separating the end of the function from the closing brace of the class.